ykrt/try_passes.py

```diff
@@ -93,7 +91,7 @@ def test_pipeline(logf, pl):
     env["PRELINK_PASSES"] = ",".join([p.name for p in pl.pre_link])
     env["LINKTIME_PASSES"] = ",".join([p.name for p in pl.link_time])
 
-    p = Popen("try_repeat 10 sh test.sh 2>&1", cwd=CWD, shell=True,
```

Comment:
is the `cwd` keyword needed? Does it not default to `pwd`?

Reply:
This is to get path to `yklua` earlier this path was hardcoded.

Reply:
do we need to change anything here?